src/main/java/frc/robot/Constants.java

```diff
@@ -1,146 +1,354 @@
+import edu.wpi.first.wpilibj2.command.Command;
+import swervelib.math.Matter;
+
+import frc.robot.commands.swervedrive.auto.*;
 
 /**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean constants. This
- * class should not be used for any other purpose. All constants should be declared globally (i.e. public static). Do
+ * The Constants class provides a convenient place for teams to hold robot-wide
+ * numerical or boolean constants. This
+ * class should not be used for any other purpose. All constants should be
+ * declared globally (i.e. public static). Do
  * not put anything functional in this class.
  *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
+ * <p>
+ * It is advised to statically import this class (or one of its inner classes)
+ * wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants
-{
-
-  public static final class ModuleConstants
-  {
-
-    public static final double kWheelDiameterMeters         = Units.inchesToMeters(
-        4); //Wheel Diameter is 4 inches - converted to meters
-    public static final double kDriveMotorGearRatio         = 6.75; //Gear Ratio
-    public static final double kTurningMotorGearRatio       = 12.8; //Gear Ratio
-    public static final double kDriveEncoderRot2Meter       = kDriveMotorGearRatio * Math.PI *
-                                                              kWheelDiameterMeters; //Takes the Gear Ratio of the
-    // multiplies it with 2PI to give you Radians
-    public static final double kDriveEncoderRPM2MeterPerSec = kDriveEncoderRot2Meter /
-                                                              60; //Divides the Drive Motor rotations in Meters by 60
-    // Driver motor and multiplies it by PI and the Diameter to get you meters
-    public static final double kTurningEncoderRot2Rad       = kTurningMotorGearRatio * 2 *
-                                                              Math.PI; //Takes the Turning Motor Gear Ratio and
-    // to give you Per Second instead of Per Min
-    public static final double kTurningEncoderRPM2RadPerSec = kTurningEncoderRot2Rad /
-                                                              60; //Divides the Turning Motor rotations in Radians by
-    // 60 to give you Per Second instead of Per Min
-    public static final double kPTurning                    = 0.5; //PID Constant P for the Turning Motor. The
-    // Proportional term already does a good enough job to tune the Motor so the "D" and "I" are not needed
+public final class Constants {
+
+  public static final double ROBOT_MASS = 48.1442941; // 106.14 Lbs
+  public static final Matter CHASSIS = new Matter(new Translation3d(0, 0, Units.inchesToMeters(8)), ROBOT_MASS);
+  public static final double LOOP_TIME = 0.13; // s, 20ms + 110ms sprk max velocity lag
+  public static final double MAX_SPEED = Units.feetToMeters(14.5);
```

Comment:
So 14.5 ft/s max velocity.
Add constants for the max acceleration, max rotational velocity, and max rotational acceleration